src/main/java/nl/tudelft/ti2206/group9/server/HighscoreServer.java

```diff
@@ -76,27 +85,33 @@ static void log(final String message) {
         public void run() {
             final Scanner sc = new Scanner(System.in, "UTF-8");
             String command;
-            log("Type \"stop\" to exit.");
             while (running) {
                 command = sc.nextLine();
-                switch (command) {
-                case "q":
-                case "stop":
-                case "exit":
-                case "evacuate":
-                case "implode":
-                    quit();
-                    try {
-                        ss.close();
-                        log("Stopping the server.");
-                    } catch (IOException e) {
-                        logError("Could not stop server? Exiting anyway :D");
+                handleCommand(command);
+            }
+            sc.close();
+        }
+
+        /**
+         * @param command the command that is entered by the user.
+         */
+        private void handleCommand(final String command) {
+            switch (command) {
+            case "q":
```

Comment:
Why also `q`? The HighScoreServer clearly tells me to enter `stop` to stop.

< review, logic